tests/test_history.py

```diff
@@ -859,6 +859,31 @@ def test_minute_dividends(self):
         # last 5 minutes should not be adjusted
         np.testing.assert_array_equal(np.array(range(782, 787)), window3[395:])
 
+    def test_passing_iterable_to_history_regular_hours(self):
+        # regular hours
+        current_dt = pd.Timestamp("2015-01-06 9:45", tz='US/Eastern')
+        bar_data = BarData(self.data_portal, lambda: current_dt, "minute")
+
+        df = pd.DataFrame(
+            [1, 2],
+            index=[self.ASSET1, self.ASSET2]
+        )
+
+        bar_data.history(df.index, "high", 5, "1m")
+
+    def test_passing_iterable_to_history_bts(self):
+        # before market hours
+        current_dt = pd.Timestamp("2015-01-07 8:45", tz='US/Eastern')
+        bar_data = BarData(self.data_portal, lambda: current_dt, "minute")
+
+        df = pd.DataFrame(
+            [1, 2],
+            index=[self.ASSET1, self.ASSET2]
```

Comment:
Could also construct this more directly as `pd.Index64([self.ASSET1, self.ASSET2])`